CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Fixed
 
  - #3054, Fix not allowing special characters in JSON keys - @laurenceisla
+ - #3090, Replace JSON parser error with generic message - @develop7
+   + Fixes #2344
```

Comment:
```suggestion
 - #2344, Replace JSON parser error with a clearer generic message - @develop7
```

Usually we link the issue directly and not the PR, unless there's no issue or the PR solves a lot of them